### PR TITLE
Home offered by agent hk

### PR DESCRIPTION
**Pull request by harshitakaushik-dev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,9 @@
     "react-native": "0.72.6",
     "react-native-gesture-handler": "~2.12.0",
     "react-native-safe-area-context": "^4.8.0",
-    "react-native-svg": "13.9.0"
+    "react-native-svg": "13.9.0",
+    "react-native-web": "~0.19.6",
```
Comment (aindrila-choudhuri) [inline package.json]: Why do we need these packages for fixing lint issues?

**Review by aindrila-choudhuri — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (harshitakaushik-dev) [reply package.json]: No, we don't need them to fix lint issues. Only added them to ease the testing. If you'd like, I can remove them.

Comment (aindrila-choudhuri) [reply package.json]: Please remove it as it is not part of this PR work.

Comment (harshitakaushik-dev) [reply package.json]: Updated, pls check.

Comment (aindrila-choudhuri) [reply package.json]: There's still an extra package in package.json. There shouldn't be any changes in package.json as part of this work.

**Review by aindrila-choudhuri — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (harshitakaushik-dev) [reply package.json]: image
I have changed it and pushed the changes. There are no pending changed from my end.

**Pull request merged**